### PR TITLE
Add configuration for the Protherm RC/2 thermostat (15.e7f_2)

### PR DESCRIPTION
**Pull request by samm-git**
This is configuration for the Protherm RC/2 device (wireless thermostat + ebus RF controller). I do not have any technical documentation, so was using my scan script to find what type of commands it supports. It is incomplete, but already allows to get and set some of the data. Sample output provided below:

```
e7f_2 DHWOverrideEndTime = 04.01.2022 01:30
e7f_2 Language = russian
e7f_2 MaintenanceAlarm = 39
e7f_2 MaintenanceAlarmDate = 2121021735
e7f_2 MaintenanceTel = ?????????????
e7f_2 RoomName = SPALNA--------
e7f_2 RoomTemp = 21.00;ok
e7f_2 Time&Date = 03.01.2022 13:35
e7f_2 VacationEndTime = 04.01.2023 13:30
e7f_2 VacationStartTime = 03.01.2023 13:29
```

and scan output:

```
$ ebusctl scan 15
15;Vaillant;E7f_2;0001;8002;21;13;24;0020118084;0082;005159;N0
```

### CODE REVIEW

File: ebusd-2.1.x/de/vaillant/15.e7f_2.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,E7f_2, Thermolink RC/2
+*r,e7f,,,,"15","B509","0D",,,,,,
+*w,e7f,,,,"15","B509","0E",,,,,,
+*wi,e7f#install,,,,"15","B509","0E",,,,,,
+
+r,,RoomTemp,,,,,"3A00",,,temp,,,Room Tempearature
```
[the remark this answers is not in the document]
Comment (samm-git) [reply ebusd-2.1.x/de/vaillant/15.e7f_2.csv]: @john30 i do knot know German, so not sure i am able to do that.

**Comment by andig** [comment]
Only thing missing is RoomTemp -> Raumtemperatur?

Viele Grüße,
Andreas

**Comment by john30** [comment]
no, the whole de file is not translated

**Comment by archont00** [comment]
There seems to be a more complete CSV file here: https://github.com/john30/ebusd-configuration/pull/232

**Comment by samm-git** [comment]
@archont00 thank you for the hint. I updated this PR with one from #232.

Changes i done:

1. Added `15.e7c` from original PR and did `15.e7f_2` as a link to it. 
2. Replaced ULG with DTM, as this data format was recently added.

Changes are tested on ebusd 21.3.p20220103 running on FreeBSD with 15.e7f_2 device. @john30 could you please merge this PR and close #232 as duplicate?  Do you need anything else to get its merged?

**Pull request merged**